### ISSUE TITLE
Notifications error:0906D06C:PEM routines:PEM_read_bio:no start line

### ISSUE THREAD

**Issue by lvarnava** [nextcloud/notifications#298]
Nextcloud 14.0.8.2
ubuntu 18.04.1
postgresql 10.6
apache 2.4.29
php 7.2.15
notifications 2.2.1
openssl 1.1.0g-2ubuntu4.3

I get an error in the logs when I add an account in the nextcloud-talk 3.2.7 (android 9) or send a message to chat or call.

nextcloud.log:
"Error	notifications	error:0906D06C:PEM routines:PEM_read_bio:no start line"
or
"error:0E06D06C:configuration file routines:NCONF_get_string:no value"
or
"error:0406D06E:rsa routines:RSA_padding_add_PKCS1_type_2:data too large for key size"

**Comment by nickvergessen** [comment]
Related logs from https://github.com/nextcloud/spreed/issues/1684 :
```json
{
  "reqId": "xJwyreNcO92VmWWyynQ6",
  "level": 4,
  "time": "2019-04-07T16:54:16+03:00",
  "remoteAddr": "xx.xxx.xxx.xx",
  "user": "username",
  "app": "notifications",
  "method": "POST",
  "url": "\/ocs\/v2.php\/apps\/spreed\/api\/v1\/chat\/ii32kaau",
  "message": "error:0906D06C:PEM routines:PEM_read_bio:no start line",
  "userAgent": "Mozilla\/5.0 (X11; Linux x86_64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/73.0.3683.103 Safari\/537.36",
  "version": "15.0.6.1"
}
{
  "reqId": "GTre9KVRKkI8dOmfVoT4",
  "level": 3,
  "time": "2019-03-12T12:48:21+03:00",
  "remoteAddr": "xx.xxx.xxx.xx",
  "user": "username",
  "app": "notifications",
  "method": "POST",
  "url": "\/ocs\/v2.php\/apps\/spreed\/api\/v1\/chat\/i62yipuq",
  "message": "error:0E06D06C:configuration file routines:NCONF_get_string:no value",
  "userAgent": "Mozilla\/5.0 (Android) Nextcloud-Talk v3.2.7",
  "version": "14.0.8.2"
}
{
  "reqId": "9pitw56HgKSqjM862lUG",
  "level": 3,
  "time": "2019-03-08T19:43:30+03:00",
  "remoteAddr": "xx.xxx.xxx.xx",
  "user": "username",
  "app": "notifications",
  "method": "POST",
  "url": "\/ocs\/v2.php\/apps\/spreed\/api\/v1\/chat\/fpj34iap",
  "message": "error:0406D06E:rsa routines:RSA_padding_add_PKCS1_type_2:data too large for key size",
  "userAgent": "Mozilla\/5.0 (Android) Nextcloud-Talk v3.2.7",
  "version": "14.0.8.2"
}
```

**Comment by thecube13** [comment]
Nextcloud 16.0.1.1
Ubuntu 18.04.2
mysql  Ver 15.1 Distrib 10.3.15-MariaDB
nginx 1.15.9
PHP 7.3.5
notifications 2.4.1
openssl 1.1.1b-1+ubuntu18.04.1+deb.sury.org+1

Nextcloud Talk android v6.0.3 (from Google play)
Call or videocall not working. Chat notifications not working.

In nextcloud.log: 
"error:0909006C:PEM routines:get_name:no start line" or 
"error:0E06D06C:configuration file routines:NCONF_get_string:no value" or 
"error:0406D06E:rsa routines:RSA_padding_add_PKCS1_type_2:data too large for key size"

**Comment by nickvergessen** [comment]
The question is what the content of the notification was, because only then we can make sure the parsed notification doesn't succeed the 1k char limit in the future anymore.

if you happen to see this message again in your logs and can guess back (from timestamp etc) which entry of oc_notifications is related, please share this with me.

**Comment by thecube13** [comment]
> The question is what the content of the notification was, because only then we can make sure the parsed notification doesn't succeed the 1k char limit in the future anymore.
> 
> if you happen to see this message again in your logs and can guess back (from timestamp etc) which entry of oc_notifications is related, please share this with me.

in my database table "oc_notifications" is missing (

**Comment by lvarnava** [comment]
Hi, nickvergessen!
I sent a text message: "zzz".

{"reqId":"zfhKcjRWHmzPkDufTDw6","level":4,"time":"2019-07-10T19:57:47+03:00","remoteAddr":"********","user":"********","app":"notifications","method":"POST","url":"/ocs/v2.php/apps/spreed/api/v1/chat/7bvuj5tf","message":"error:0909006C:PEM routines:get_name:no start line","userAgent":"Mozilla/5.0 (Android) Nextcloud-Talk v6.0.7","version":"15.0.10.0","id":"5d2619bc149c4"}

**Comment by nickvergessen** [comment]
also the `data too large for key size` error?

**Comment by nickvergessen** [comment]
Should be fixed with #383  as well
